.github/workflows/release-github.yaml

```diff
@@ -0,0 +1,84 @@
+name: release
+on:
+  release:
+    types: [published]
+env:
+  NAME: "grafana-operator"
+jobs:
+  helm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v3
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+      - name: Get GitHub Tag
+        id: get_tag
+        run: |
+          echo "tag=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
+      - name: Publish Helm charts
+        run: |
+          cd helm
+          helm registry login -u ${{ github.repository_owner }} -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
+          helm package --app-version ${{ steps.get_tag.outputs.tag }} --version ${{ steps.get_tag.outputs.tag }} grafana-operator
+          helm push grafana-operator-${{ steps.get_tag.outputs.tag }}.tgz oci://ghcr.io/grafana-operator/helm-charts
+  image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v3
+      - name: Prepare
+        id: prep
+        run: |
+          VERSION=sha-${GITHUB_SHA::8}
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF/refs\/tags\//}
+          fi
+          echo "BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_OUTPUT
+          echo "VERSION=${VERSION}" >> $GITHUB_OUTPUT
+      - name: Get GitHub Tag
+        id: get_tag
+        run: |
+          echo "tag=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
+      - name: Cache container layers
+        uses: actions/cache@v2.1.4
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Set up QEMU
```

Comment:
QEMU is slow as hell when trying to build an image for other architectures. When I tested it for a tiny app with not much dependencies, it took 4m28s instead of 51s. With the operator, it's likely to be much more.
The fastest option is to build an image for different architectures on the host and then copy binaries. I think `go-releaser` or `ko` would be much better in that sense. WDYT?

Reply:
Saw now that my build of these images in my fork took 40 minutes to run....

Reply:
Yeah, that's what I was expecting to see :)
Unfortunately, I don't have a ready-to-use example, it's something to dig into.

I guess `go-releaser` is the most flexible and functional option nowadays, it's supports cosign (since you mentioned it) and so many different options. We can either try to go through their documentation or look at how other projects are using it.

Reply:
I can see if I get time to take a look at it today. We could start to merge this PR and then we can work on making it faster, we don't do releases that often so it shouldn't be a big issue. I will also check a bit on our old setup because it don't take 40 min...

Reply:
I don't get it, our old repo it only took about 6 min to build the container images: https://github.com/grafana-operator/grafana-operator/actions/runs/4142896346/jobs/7164110490
Sure there are some differences between the setups but it's more or less the same...